release_notes/4.2-notes.rst

```diff
@@ -31,3 +31,18 @@ Additions of PyBuiltins
 The `PyBuiltins <http://ninia.github.io/jep/javadoc/4.2/jep/python/PyBuiltin.html>`_
 class provides makes it easier to call some of the builtin Python functions
 from Java.
+
+Java FunctionalInterfaces are automatically callable in Python 
+**************************************************************
+When a Java object implementing a
+`FunctionalInterface <https://docs.oracle.com/en/java/javase/21/docs/api/java.base/java/lang/FunctionalInterface.html>`_
+is used in Python it is automatically 
+`callable <https://docs.python.org/3/glossary.html#term-callable>`_
+and the `__call__` method will be set to the abstract method of the interface.
+Similar to the Java compiler, the type is not required to have the
+FunctionalInterface annotation to be treated as a functional interface.
+For example a Kava Function can be called directly without using the accept method:
```

Comment:
Did you mean Java where you wrote Kava?

Reply:
Yes thanks, fixed nnow. I am pretty sure I was correcting capitalization and inadvertently changed java to Kava.